### PR TITLE
fix(batch): fix batch script + add example

### PR DESCRIPTION
**Pull request by mvaullerin**

### CODE REVIEW

File: src/core/wallet/services/batchService/batchService.ts [hunk 1]
```diff
@@ -22,48 +21,46 @@ export class BatchService {
...
         txRow: transaction.rawTransaction,
         creationDate: new Date()
-      });
-
-      batch.add(this.web3Service.web3.eth.sendSignedTransaction(transaction.rawTransaction));
+      };
     });
 
+    const transactions = await Promise.all(transactionsPromise);
     this.batchTransactions = [];
```
Comment (stefdelec) [inline src/core/wallet/services/batchService/batchService.ts]: is this class property sill used?

File: src/core/wallet/services/batchService/batchService.ts [hunk 2]
```diff
@@ -22,48 +21,46 @@ export class BatchService {
...
-    batch.execute();
-    return this.watchPendingTransaction();
+    return this.watchPendingTransaction(transactions);
   }
 
-  public watchPendingTransaction () {
+  public watchPendingTransaction (transactionInProgress) {
     return new Promise((resolve, reject) => {
```
Comment (stefdelec) [inline src/core/wallet/services/batchService/batchService.ts]: I think this method never resolves and interval is never cleared

File: src/core/wallet/services/batchService/batchService.ts (hunk 1 above)
Comment (mvaullerin) [reply src/core/wallet/services/batchService/batchService.ts]: Yes, 2 line below

File: src/core/wallet/services/batchService/batchService.ts (hunk 2 above)
Comment (mvaullerin) [reply src/core/wallet/services/batchService/batchService.ts]: check line 68 and above

**Pull request merged**